exotic/api/colab.py

```diff
@@ -109,7 +109,7 @@ def display_image(filename):
     # )
 
     # create a figure with text on mouse hover
-    fig = figure(tooltips=[("x", "$x"), ("y", "$y"), ("value", "@image")], plot_width=p_width, plot_height=p_height,
+    fig = figure(tooltips=[("x", "$x"), ("y", "$y"), ("value", "@image")], width=p_width, height=p_height,
```

Comment:
@jpl-jengelke - should this be a try/except instead to ensure backwards compatibility with the previous version of bokeh?

Reply:
Well, I thought no. Since it will be tagged as 4.0.1, we just need to make sure that it is only used to accompany a new Colab notebook. So releasing an upgraded notebook would be the solution.